src/Streamly/Internal/Data/Stream/IsStream.hs

```diff
@@ -2201,11 +2201,13 @@ scanx = P.scanlx'
 -- complexity when iterated over a stream. We should use StreamK style scanlM'
 -- for linear performance on iteration.
 --
--- | Like 'scanl'' but with a monadic fold function.
+-- | Like 'scanl'' but with a monadic step function and a monadic seed.
 --
--- @since 0.4.0
+-- /Since: 0.4.0/
+--
+-- /Since: 0.8.0 (signature change)/
```

Comment:
I think I changed almost everywhere, when `Internal.Prelude` to `IsStream` change has been made, to have the latest _Since_ annotation to be the last. Also if we use `@since ` (the one haddock provides) it always gets rendered at the bottom of the doc.